includes/CLI/Plugin_Check_Command.php

```diff
@@ -0,0 +1,312 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\CLI\Plugin_Check_Command
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\CLI;
+
+use WordPress\Plugin_Check\Plugin_Context;
+use WordPress\Plugin_Check\Utilities\Plugin_Request_Utility;
+use WordPress\Plugin_Check\Checker\CLI_Runner;
+use Exception;
+use WP_CLI;
+
+/**
+ * Plugin check command.
+ */
+class Plugin_Check_Command {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since n.e.x.t
+	 * @var Plugin_Context
+	 */
+	protected $plugin_context;
+
+	/**
+	 * Output format type.
+	 *
+	 * @since n.e.x.t
+	 * @var string[]
+	 */
+	protected $output_formats = array(
+		'table',
+		'csv',
+		'json',
+	);
+
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Plugin_Context $plugin_context Plugin context.
+	 */
+	public function __construct( Plugin_Context $plugin_context ) {
+		$this->plugin_context = $plugin_context;
+	}
+
+	/**
+	 * Run plugin check.
+	 *
+	 * ## OPTIONS
```

Comment:
@jjgrainger Is the options argument working for you? or will it be in an upcoming PR?

`npm run wp-env run cli wp plugin check akismet --format=json` it give me an error `Error: Too many positional arguments: json`

Reply:
Thanks @mukeshpanchal27 I experienced the same issue here but am uncertain what the cause is. I noticed that the docblock comment for the command wasn't correct for these options, however updating these has not resolved the issue. I'll keep looking into this.

Reply:
Since you're invoking WP-CLI in the container, I think you're missing an extra `--`. You need to use `npm run wp-env run cli -- wp plugin check akismet --format=json`